### PR TITLE
LRT-4077: Move Live Chat logic from Communication Service

### PR DESCRIPTION
**Pull request by nikolaydanekov**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -39,10 +39,10 @@
         <jacoco-maven-plugin.version>0.8.4</jacoco-maven-plugin.version>
         <github.global.server>github</github.global.server>
         <exists-maven-plugin.version>0.4.0</exists-maven-plugin.version>
-        <spring-boot.version>2.2.5.RELEASE</spring-boot.version>
+        <spring-boot.version>2.5.6.RELEASE</spring-boot.version>
         <!--		Please, update starters to the latest versions		-->
```
Comment (istoeva) [inline pom.xml:43]: Remove unnecessary comments

File: .circleci/config.yml [hunk 2]
```diff
@@ -33,8 +33,7 @@ workflows:
       - deployment-tools/java-build:
           context: deployment-dev
           build-image: circleci/openjdk:11.0.8-jdk-buster
-#         Set "deploy: true" if deployment to the github packages is required
-          deploy: false
+          deploy: true
```
Comment (istoeva) [inline .circleci/config.yml:36]: you won't need a deployment to the github packages

File: README.md [hunk 3]
```diff
@@ -16,12 +15,14 @@ How to run project locally
     client.cis.basic_auth_username=cisuser
     client.cis.basic_auth_password=<password>
     ```
-- Run `JavaServiceTemplateStarterApplication.java` class
-
+- Run `JavachatStarterApplication.java` class
```
Comment (istoeva) [inline README.md:18]: Update the class name

File: lendage-chat-service-main/src/main/java/com/lendage/chat/client/LiveChatClient.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+import lombok.extern.log4j.Log4j2;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.web.client.HttpStatusCodeException;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+
+@Log4j2
```
Comment (istoeva) [inline lendage-chat-service-main/src/main/java/com/lendage/chat/client/LiveChatClient.java:19]: Remove unused annotation

**Review by Ivan-Tashev — APPROVED** [review]

**Review by istoeva — APPROVED** [review]

**Pull request merged**